packages/components/checkbox/src/use-checkbox-group.ts

```diff
@@ -14,7 +14,7 @@ import {clsx, safeAriaLabel} from "@nextui-org/shared-utils";
 
 import {CheckboxProps} from "./index";
 
-interface Props extends HTMLNextUIProps<"div"> {
+interface Props extends Omit<HTMLNextUIProps<"div">, "onChange"> {
```

Comment:
As indicated by the warning, the issue resides with `FormEventHandler<HTMLDivElement>`. Therefore, I believe the following code should be corrected.
https://github.com/nextui-org/nextui/blob/main/packages/components/checkbox/src/use-checkbox-group.ts#L49

I think it would suffice to simply remove `onChange` from `Props`.
```
export type UseCheckboxGroupProps = Omit<Props, "onChange"> &
```

This would eliminate the need to redefine `onChange`.

Reply:
Thanks. much cleaner.